src/main/resources/templates/geoweaver.html

```diff
@@ -87,24 +87,32 @@
 	    
 	    <script type="text/javascript" src="../js/no-queue.js"></script>
 		
-		<script type="text/javascript" src="../js/codemirror/lib/codemirror.js"></script>
```

Comment:
can you download these 3rd party javascript files to Geoweaver repo? Geoweaver might need to work on machines without Internet. Make sure not external links are necessary to render the GUI.